packages/playground/website/src/components/terminal-button/index.tsx

```diff
@@ -0,0 +1,273 @@
+import Modal from 'react-modal';
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import css from './style.module.css';
+import {PlaygroundClient} from '@wp-playground/playground-remote';
+import {Terminal} from 'xterm';
```

Comment:
64kb uncompressed, maybe import asynchronously? Vite will handle code splitting when it finds `import('xterm')`